projects/kit/components/input-date-multi/input-date-multi.template.html

```diff
@@ -0,0 +1,76 @@
+<tui-hosted-dropdown
+    class="t-hosted"
+    [canOpen]="canOpen"
+    [content]="dropdown"
+    [open]="open && canOpen"
+    (openChange)="onOpenChange($event)"
+>
+    <tui-input-tag
+        *tuiLet="filler$ | async as filler"
```

Comment:
Unnecessary, can be just `[tuiTextfieldFiller]="(filler$ | async) || ''"`

Reply:
fixed